### DISCUSSION TITLE
RAG and Fabric

### DISCUSSION THREAD

**Discussion by quietrags** [danielmiessler/Fabric#329] [Ideas]
Two question for the community. Maybe these are features that are already there but i could not find them, Do point me to discussions in case this is already there. 

1. Is there a way one could combine documents with information as additional context to patterns like write_essay or create_keynote?

2. Is there a way one could give do a simple Q&A with a document using fabric?

Thanks

**Comment by djrocket** [comment]
had the same questions...  getting RAG started would be awesome, at least the "naive" version

**Reply by nic0711** [reply]
Maybe it is possible to use something like "anything llm"?

**Reply by profucius** [reply]
The RAG feature in Msty.app is superior (it's called Knowledge Stacks there)
